apps/data-pipeline/ap-exams-importer/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@apps/ap-exams-importer",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Script for importing AP Exam names from a statuc mapping",
```

Comment:
```suggestion
  "description": "Script for importing AP exam names from a static mapping",
```

Reply:
Typo, indeed, but "AP Exam" is the capitalization used by College Board.